src/styles/main.scss

```diff
@@ -1,3 +1,32 @@
+@import "./variables.scss";
+@import "./header.scss";
+@import "./catalog.scss";
+@import "./card.scss";
+@import "./stars.scss";
+@import "./adaptive.scss";
+
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
```

Comment:
Add the fallback font-family
![image](https://github.com/mate-academy/layout_catalog/assets/63243136/36759360-6f5c-4714-bb1a-012de258dfe7)